crates/holochain/benches/consistency.rs

```diff
@@ -157,7 +157,7 @@ async fn setup() -> (Producer, Consumer, Others) {
     let alice = cells.next().unwrap();
     let bobbo = cells.next().unwrap();
 
-    conductors.exchange_peer_info().await;
+    //conductors.exchange_peer_info().await;
```

Comment:
I've left these in for now in case we need to come up with something equivalent in k2.